src/main/java/racinggame/business/Process.java

```diff
@@ -1,7 +1,21 @@
 package racinggame.business;
 
+import racinggame.model.*;
 import racinggame.response.GameResponse;
 
-public interface Process {
-    public GameResponse run(String[] names, int moves) ;
-}
+public class Process {
+
+    private String[] names;
+    private final Move move;
+
+    public Process(Move move, String[] names) {
+        this.names = names;
+        this.move = move;
+    }
+
+    public GameResponse run() {
+        JoinCars joinCars = CarFactory.createCars(names);
+        GameResult gameResult = move.moveCar(joinCars);
+        return new GameResponse(gameResult, gameResult.findLastRaceRecord().findGameWinners().WinnerCarsName());
+    }
```

Comment:
```suggestion
    public GameResult run() {
        JoinCars joinCars = CarFactory.createCars(names);
        GameResult gameResult = move.moveCar(joinCars);
        return gameResult;
    }
```
위와 같이 GameResult만 반환해도 되지 않을까?
사용하는 곳에서는

```java
GameResult result = ...
ProcessRecord record = result.getProcessRecord(0//index);
ProcessRecord lastRecord = result.getLastRecord();
List<Car> winnersName =  lastRecord.winnersName();
```
이와 같이 접근해 가면 어떨까?